requirements/ci-wheel.txt

```diff
@@ -2,7 +2,7 @@
 attrs==19.1.0
 async-generator==1.10
 async-timeout==3.0.1
-brotlipy==0.7.0
+Brotlipy==1.0.7
```

Comment:
It seems should be just `Brotli==1.0.7`, right?

Reply:
Indeed, fixed!